Add unit test: text-classification pipeline handles empty string

ckakgun opened the pull request:
## What does this PR do?

This  pull request adds a unit test to the `text-classification` pipeline to verify that it can handle an empty string input without raising an error, and returns an empty list. This improves test coverage for an edge case.

## Motivation

Sometimes a user fills empty input into a pipeline by mistake.  This test ensures that the pipelines deal with the issue of having an empty string.

ckakgun committed:
Add unit test: text-classification pipeline handles empty string

Rocketknight1 commented:
hi @ckakgun is this something the pipeline is failing at right now, or does the test just verify behaviour that already works?

ckakgun committed:
Fix newline issue in text classification pipeline test for empty string handling

ckakgun commented:
Hi @Rocketknight1! thanks for the reply.
This test verifies a behavior that already works. It aims to improve test coverage for an edge case (empty input) to ensure stability.

ckakgun committed:
Apply Black formatting to test_pipelines_text_classification.py

- Fixed code formatting issues identified by Black formatter
- No functional changes to the test logic
- Maintains original test expectations: empty string should return empty list

Rocketknight1 commented:
Hmmn - I'm not sure the test is needed. I don't know if we need defined behaviour in the case when the pipeline is passed an empty string. since it's not clear to me what the user's expectations should be there! It also seems like something we might break/change in future, and I don't want the test to make any guarantees about what happens in case they become invalid later.

Still, thank you for the PR! Feel free to keep contributing in future.
